pkg/cli/log.go

```diff
@@ -17,16 +18,24 @@ func NewLogs() *cobra.Command {
 }
 
 type Logs struct {
-	Follow bool `short:"f" usage:"Follow log output"`
+	Follow    bool  `short:"f" usage:"Follow log output"`
+	TailLines int64 `short:"n" usage:"Number of lines in log output"`
```

Comment:
Out of curiosity why do you explicitly need a int64 over just `int`?  Just that `int` was an already supported type.

Reply:
It was already defined as an int64 in the code. Looks like from k8s definition located at ~/go/pkg/k8s.io/api@v0.25.2/core/v1/types.go line 5504

Reply:
It was already defined as an int64 in the code. Looks like from k8s definition located at ~/go/pkg/k8s.io/api@v0.25.2/core/v1/types.go line 5504